packages/block-directory/src/store/actions.js

```diff
@@ -54,16 +55,35 @@ export function* installBlockType( block ) {
 			throw new Error( __( 'Block has no assets.' ) );
 		}
 		yield setIsInstalling( block.id, true );
-		const response = yield apiFetch( {
-			path: 'wp/v2/plugins',
-			data: {
-				slug: block.id,
-				status: 'active',
-			},
-			method: 'POST',
+
+		// If we have a wp:plugin link, the plugin is installed but inactive.
+		const url = getPluginUrl( block );
+		let links = {};
+		if ( url ) {
+			yield apiFetch( {
+				url,
+				data: {
+					status: 'active',
+				},
+				method: 'PUT',
+			} );
+		} else {
+			const response = yield apiFetch( {
```

Comment:
Not necessary, but we could also use the `wp:install-plugin` link as a base, and just add the `status: active` data.